sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1254,6 +1254,30 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("CREATE TABLE LIKE") {
+    val targetTable = "testcat.ns1.ns2.tbl1"
+    val sourceTable = "testcat.ns1.ns2.tbl2"
+    withTable(targetTable, sourceTable) {
+      val e1 = intercept[AnalysisException] {
+        sql(s"CREATE TABLE $targetTable LIKE $sourceTable")
+      }
+      assert(e1.message.contains("CREATE TABLE LIKE is only supported with v1 tables"))
+
+      val e2 = intercept[AnalysisException] {
+        sql(s"CREATE TABLE IF NOT EXISTS $targetTable LIKE $sourceTable")
+      }
+      assert(e2.message.contains("CREATE TABLE LIKE is only supported with v1 tables"))
+
+      withTempDir { tmpDir =>
```

Comment:
Just a question, do we need to create `withTempDir`?

Reply:
@dongjoon-hyun Actually its not required. I have removed it.